test/gtest/uct/test_ib.cc

```diff
@@ -168,4 +169,74 @@ UCS_TEST_P(test_uct_ib, non_default_pkey, "IB_PKEY=0x2")
     free(recv_buffer);
 }
 
+UCS_TEST_P(test_uct_ib, wakeup, "IB_TX_CQ_MODERATION=0")
```

Comment:
pls make this test for all transports. also TX signaling should tell when there are new send resources. It should not require setting moderation=0 to work.